openslides/motions/static/templates/motions/motion-detail/personal-note.html

```diff
@@ -14,6 +14,12 @@
           <translate>Inline editing</translate>
         </button>
       </div>
+      <div class="pull-right inline-editing-activator">
+        <button ng-click="exportPersonalNote()" class="btn btn-default btn-sm">
+          <i class="fa fa-file-pdf-o"></i>
+          <translate>Export personal note</translate>
```

Comment:
Please use just "PDF" as button text (like the motion pdf button) and add these string ("Export personal note") as tooltip.